app/views/exports/bioscan_mbrave.tsv.erb

```diff
@@ -0,0 +1,26 @@
+<%= 
+    aliquot = @tube.aliquots.first    
+    first_tag_1_group_name = aliquot.tag.tag_group.name
+    version = mbrave_tag_version(first_tag_1_group_name)
+
+    rows = [['Forward Labels', 'Reverse Labels', 'Label', 'Group', 'UMI plate ID', 'Sample Plate ID']]
+    @tube.aliquots.each do |aliquot| 
+      tag_1_group_name = aliquot.tag.tag_group.name
+      tag_2_group_name = aliquot.tag2.tag_group.name
+
+      if (version != mbrave_tag_version(tag_1_group_name)) || (version != mbrave_tag_version(tag_2_group_name))
+        raise "This tube contains tags from different tag groups"
```

Comment:
..'from different tag group versions'

(the tube will always contain tags from different tag groups)

Reply:
I thought it was going to be 2 tag groups: tag_1 and tag_2, and then we check that all the 9600 aliquots are using the same tag groups (the same version).